src/Components/Facility/FacilityOxygenInformation.tsx

```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import { OXYGEN_TYPES } from "../../Common/constants";
+import { NonReadOnlyUsers } from "../../Utils/AuthorizeFor";
+import DialogModal from "../Common/Dialog";
+import ButtonV2 from "../Common/components/ButtonV2";
+import CareIcon from "../../CAREUI/icons/CareIcon";
+import { OxygenDetails } from "./OxygenDetails";
+import { OxygenTypeCard } from "./OxygenTypeCard";
+
+export const FacilityOxygenInformation = ({
+  facilityId,
+  facilityData,
+  facilityFetch,
+}: any) => {
+  const [oxygenDetailsModalOpen, setOxygenDetailsModalOpen] = useState(false);
+
+  let capacityList: any = null;
```

Comment:
Will "let capacityList: ReactElement | null = null" be good? Since it will contain React element(reference line 25 or Line 38)
Moreover this file is almost in similar pattern with and FacilityBedCapacity.tsx. Should there also this variable be modified?